banca-noix/src/main/java/com/basis/bsb/bancanoix/servico/dto/CargoDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.basis.bsb.bancanoix.servico.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
se vc não for criar um construtor para esta classe, o java já cria um construtor vazio, então não vai precisar do no args